javascripts/content_scripts/privly.js

```diff
@@ -299,8 +298,8 @@ var privly = {
       var notUpdated = [];
       elements.forEach(
         function(a){
-          if ( ! a.textContent.indexOf("privlyInject1") > 0 // Optimization
-            || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
+          // Optimization
+          if (  a.textContent.indexOf("privlyInject1") < 0 || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
```

Comment:
I think the best thing to do here is to move each of the boolean statements into their own local variable. Something roughly like:

`var taggedForInjection = ...`  
`var copiedOver = taggedForInjection || ! privly.correctIndirection.testAndCopyOver(a, a.textContent)`

then the if statement.

I would also be OK with dropping this change and letting someone concentrate on this in the future for their "level 0." You have other changes here.